care.config.ts

```diff
@@ -38,6 +38,9 @@ const careConfig = {
   customLogo: logo(env.REACT_CUSTOM_LOGO),
   customLogoAlt: logo(env.REACT_CUSTOM_LOGO_ALT),
   customDescription: env.REACT_CUSTOM_DESCRIPTION,
+  availableLocales: (env.REACT_AVAILABLE_LOCALES || "")
+    .split(",")
+    .map((l) => l.trim().toLowerCase()),
```

Comment:
```suggestion
    .map((l) => l.trim()),
```

Reply:
@bodhish this is to be sure that random case characters in the ENV don't break things.

Reply:
Lets not have it, env configuration is one time and validation for the env will take care of it. https://github.com/ohcnetwork/care_fe/issues/8470